component-library/src/app/components/title-slug-url/title-slug-url.component.scss

```diff
@@ -4,12 +4,18 @@
 }
 
 .h1-heading-type {
+  h1 {
+    margin: 0px;
+  }
   margin: {
     top: 48px;
     bottom: 24px;
   }
 }
 .h2-heading-type {
+  h2 {
+    margin: 0px;
+  }
   margin: {
     top: 80px;
```

Comment:
For `Overview`, `for developers`, `for designers` pages this margin is `64px`

Reply:
Will check with design to make it consistent across pages, since it is 80px for all the component pages

Reply:
Design got back to me and they said it should be 80px everywhere